src/client/index.js

```diff
@@ -1,59 +1,54 @@
 const http = require('http');
 const { TimeoutError, BadServerDataError } = require('../errors');
-const { gererateID } = require('../util');
+const { generateID } = require('../util');
 
 const defaultOptions = {
     host: '127.0.0.1',
     port: 6356,
-    timeout: 0
+    path: '/',
+    connectTimeout: 0,
+    responseTimeout: 0
 };
 
 class Client {
     constructor(userOptions) {
         this.options = Object.assign({}, defaultOptions, userOptions || {});
+        this.options.headers = Object.assign({}, this.options.headers || {}, { 'Content-Type': 'application/json' });
```

Comment:
I see no reason to disallow a custom `Content-Type`. Even if I see no reason to change it, I think it should be allowed 🙂